src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  function getSum(x) {
+    return function(y) {
+      count++;
+
+      if (count === 4) {
+        return 'Bzzz... Error!';
+      }
+
+      return x + y;
+    };
+  };
+
+  return getSum;
 }
```

Comment:
Good job, but your code doesn't work correctly.
Your program passed all tests, but the test missed the main request of the task.
Look,  Then all even calls return the string `Bzzz... Error!` 
<img width="363" alt="Screenshot 2023-03-24 at 11 36 57" src="https://user-images.githubusercontent.com/106397016/227482658-7089d6c2-0833-4e80-bc73-3fa9ddd1586c.png">
but in your program when I started your program I did receive not the correct answer.
<img width="438" alt="Screenshot 2023-03-24 at 11 42 01" src="https://user-images.githubusercontent.com/106397016/227484090-c4d7e8c7-9ed9-486a-97a6-503a99e47e86.png">

Reply:
fixed